common/src/stack/command/stack/commands/load/__init__.py

```diff
@@ -149,7 +149,7 @@ def load_partition(self, partitions, target=None):
 				  'partid'    : p.get('partid'),
 				  'mountpoint': p.get('mountpoint'),
 				  'size'      : p.get('size'),
-				  'fstype'    : p.get('fstype'),
+				  'type'    : p.get('fstype'),
```

Comment:
alignment's f'd up, yo.  You're gonna wear down that left front tire a lot faster.